polkadot/node/service/src/lib.rs

```diff
@@ -979,11 +980,24 @@ pub fn new_full<
 		let fetch_chunks_threshold =
 			if config.chain_spec.is_polkadot() { None } else { Some(FETCH_CHUNKS_THRESHOLD) };
 
+		let availability_config = AvailabilityConfig {
+			col_data: parachains_db::REAL_COLUMNS.col_availability_data,
+			col_meta: parachains_db::REAL_COLUMNS.col_availability_meta,
+			keep_finalized_for: if matches!(
+				config.chain_spec.identify_chain(),
+				Chain::Kusama | Chain::Polkadot | Chain::Westend
+			) {
+				KEEP_FINALIZED_FOR_LIVE_NETWORKS
+			} else {
+				keep_finalized_for.unwrap_or(1)
```

Comment:
Why not match only rococo above ? This would keep 25h interval for anything else, including forks of Polkadot.